net-p2p/soulseek-qt/soulseek-qt-20180130.ebuild

```diff
@@ -1,33 +1,37 @@
-# Copyright 1999-2016 Gentoo Foundation
+# Copyright 2019 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=5
+EAPI=7
+
+inherit desktop
 
 DESCRIPTION="Official binary Qt SoulSeek client"
 HOMEPAGE="http://www.soulseekqt.net/"
 BINARY_NAME="SoulseekQt-${PV:0:4}-$((${PV:4:2}))-$((${PV:6:2}))"
-SRC_URI="
-	x86? ( https://www.dropbox.com/s/kebk1b5ib1m3xxw/${BINARY_NAME}-32bit.tgz )
-	amd64? ( https://www.dropbox.com/s/7qh902qv2sxyp6p/${BINARY_NAME}-64bit.tgz )
-"
+SRC_URI="https://www.slsknet.org/SoulseekQt/Linux/${BINARY_NAME}-64bit-appimage.tgz https://www.dropbox.com/s/3qdatnoi10e3c17/slsk.png"
 
 LICENSE="free-noncomm"
```

Comment:
This is wrong, see [bug 702334](https://bugs.gentoo.org/702334#c3).

Before we can add this, we at least need some statement from upstream about the bundled GPL libraries. Why do they think that this isn't a GPL violation?